lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,83 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+          line_of_code: integer?,
+          last_commit_datetime: string
+      )
+    end
+
+    FILE_COMMAND_VALID_PLAIN_TEXT_FORMATS = ["ASCII text", "UTF-8 Unicode text", "ISO-8859 text"].freeze
+
+    def analyzer_version
+      @analyzer_version ||= capture3!("wc", "--version").then do |stdout, |
+        stdout.split(/\R/)[0][/\d*\.\d*/].then {|ver| ver.nil? ? "unknown" : ver}
+      end
+    end
```

Comment:
Can you use useful `extract_version!` method of `Processor` class as?

```suggestion
    def analyzer_version
      @analyzer_version ||= extract_version!("wc")
    end
```

Reply:
fix: 3ee11d1f329743ce85acba5da3234c7c49dab038